src/test/scala/mesosphere/mesos/ResourceMatcherTest.scala

```diff
@@ -30,13 +34,11 @@ class ResourceMatcherTest extends MarathonSpec with Matchers {
     resOpt should not be empty
     val res = resOpt.get
 
-    res.cpuRole should be("*")
-    res.memRole should be("*")
-    res.diskRole should be("")
+    res.scalarMatch(Resource.CPUS).get.roles should be(Seq("*"))
+    res.scalarMatch(Resource.MEM).get.roles should be(Seq("*"))
+    res.scalarMatch(Resource.DISK).get.roles should be(Seq.empty)
 
-    // check if we got 2 ports
-    val range = res.ports.head.ranges.head
-    (range.end - range.begin) should be (1)
+    res.hostPorts should have size 2
```

Comment:
the previous test code also tested that the ports are subsequent numbers – not the responsibility of this test, probably there's another test that verifies this …?